tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -474,10 +474,13 @@ private static void testMouseWheel(int amount) {
         InvalidationListener invalidationListener = observable -> setSceneLatch.countDown();
         int[] totalScroll = new int[]{0};
         long[] firstScrollMillis = new long[]{0};
+        // The scroll wheel amount is multiplied by 40 on Linux and Windows, but not on macOS. The
+        // directions are also reversed. This difference in behavior is unexplained and might be a bug.
+        // See JDK-8214580.
         int scrollMultiplier = System.getProperty("os.name").toLowerCase().startsWith("mac") ? -1 : 40;
         Util.runAndWait(() -> {
             button.setOnScroll(event -> {
-                totalScroll[0] += event.getDeltaY();
+                totalScroll[0] += event.getDeltaY() * Screen.getPrimary().getOutputScaleY();
```

Comment:
Multiplying by the output scale works correctly on Windows. It doesn't work on a Mac with a retina display, though. The test now fails for me. This suggests another platform difference between how Robot::mouseWheel works on Mac versus the other platforms.

Reply:
Do you think we should skip the test on Mac for now and un-skip it as part of JDK-8214580?

Reply:
Okay, I added a new commit and updated the bug.